[2025-03-12T18:07:10.428+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-03-12T18:07:10.442+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: data_lake_pipeline.start_pipeline manual__2025-03-12T18:07:06.905107+00:00 [queued]>
[2025-03-12T18:07:10.451+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: data_lake_pipeline.start_pipeline manual__2025-03-12T18:07:06.905107+00:00 [queued]>
[2025-03-12T18:07:10.451+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 2
[2025-03-12T18:07:10.475+0000] {taskinstance.py:2890} INFO - Executing <Task(BashOperator): start_pipeline> on 2025-03-12 18:07:06.905107+00:00
[2025-03-12T18:07:10.482+0000] {standard_task_runner.py:72} INFO - Started process 223 to run task
[2025-03-12T18:07:10.496+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'data_lake_pipeline', 'start_pipeline', 'manual__2025-03-12T18:07:06.905107+00:00', '--job-id', '48', '--raw', '--subdir', 'DAGS_FOLDER/data_pipeline_dag.py', '--cfg-path', '/tmp/tmp8nrwur2u']
[2025-03-12T18:07:10.498+0000] {standard_task_runner.py:105} INFO - Job 48: Subtask start_pipeline
[2025-03-12T18:07:10.530+0000] {logging_mixin.py:190} WARNING - /usr/local/lib/python3.10/site-packages/***/settings.py:209 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-03-12T18:07:10.599+0000] {task_command.py:467} INFO - Running <TaskInstance: data_lake_pipeline.start_pipeline manual__2025-03-12T18:07:06.905107+00:00 [running]> on host ***-scheduler
[2025-03-12T18:07:10.739+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='data_lake_pipeline' AIRFLOW_CTX_TASK_ID='start_pipeline' AIRFLOW_CTX_EXECUTION_DATE='2025-03-12T18:07:06.905107+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-03-12T18:07:06.905107+00:00'
[2025-03-12T18:07:10.743+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-03-12T18:07:10.769+0000] {subprocess.py:78} INFO - Tmp dir root location: /tmp
[2025-03-12T18:07:10.770+0000] {subprocess.py:88} INFO - Running command: ['/usr/bin/bash', '-c', 'echo "Pipeline démarré"']
[2025-03-12T18:07:10.785+0000] {subprocess.py:99} INFO - Output:
[2025-03-12T18:07:10.787+0000] {subprocess.py:106} INFO - Pipeline démarré
[2025-03-12T18:07:10.788+0000] {subprocess.py:110} INFO - Command exited with return code 0
[2025-03-12T18:07:10.830+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-03-12T18:07:10.831+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=data_lake_pipeline, task_id=start_pipeline, run_id=manual__2025-03-12T18:07:06.905107+00:00, execution_date=20250312T180706, start_date=20250312T180710, end_date=20250312T180710
[2025-03-12T18:07:10.876+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-03-12T18:07:10.916+0000] {taskinstance.py:3901} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-03-12T18:07:10.924+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
